content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -16,7 +16,17 @@ further_reading:
 ---
 
 <div class="alert alert-danger">
-This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups. Use a <strong>Pull</strong> subscription with the Datadog Dataflow template to forward your Google Cloud logs to Datadog instead. See <a href="https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection" target="_blank">Log collection</a> on the Google Cloud integration page for instructions.
+
+This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups.
+
+Collecting Google Cloud logs with a Pub/Sub Push subscription is in the process of being deprecated for the following reasons:
+
+- If you have a Google Cloud VPC, new Push subscriptions cannot be configured with external endpoints (see Google Cloud's [Supported products and limitations][12] page for more information)
+- The Push subscription does not provide compression or batching of events, and as such is only suitable for a very low volume of logs
```

Comment:
Do customers ask the reasoning for deprecation? I think we can remove this. I mainly ask because this is a very long warning note:
![image](https://github.com/user-attachments/assets/31f4b00b-625d-410f-a5a7-8a7538686981)

Reply:
it speaks to the value proposition and customers do ask why. I am open to making it more succinct. cc @buraizu